src/VisualStudio/Core/Def/Implementation/Preview/FileChange.cs

```diff
@@ -44,6 +45,7 @@ internal class FileChange : AbstractChange
             _left = left;
             _right = right;
             _imageService = imageService;
+            _isAddionalDocument = !((left ?? right) is Document);
```

Comment:
FileChange actually accepts TextDocument, is there a reason it can't handle additional document which should be text document?

also, assuming a document as an additional document just because it is not Document seems too fragile.